packages/hop-node/src/theGraph/getTransferRoots.ts

```diff
@@ -30,23 +34,19 @@ export default async function getTransferRoots (chain: string, token: string, sk
   `
   const jsonRes = await makeRequest(chain, query, {
     token,
-    skip
+    lastId
   })
 
   let roots = jsonRes.transfersCommitteds.map((x: any) => normalizeEntity(x))
 
-  if (roots.length === 1000) {
-    try {
-      roots = roots.concat(await getTransferRoots(
-        chain,
-        token,
-        skip + 1000
-      ))
-    } catch (err) {
-      if (!err.message.includes('The `skip` argument must be between')) {
-        throw err
-      }
-    }
+  lastId = roots[roots.length - 1].id
```

Comment:
is it possible that `roots.length` here can be `0` so `.id` would throw an error trying to read from `undefined`?

Reply:
Yes, great call. Fixed with 26ffbf5f93e5a0d1cd10ed6b2e87d8b1919e4177